backend/app/user/routes.py

```
@@ -55,3 +66,62 @@ async def delete_user_account(current_user: Dict[str, Any] = Depends(get_current
     return DeleteUserResponse(
         success=True, message="User account scheduled for deletion."
     )
+
+
+@router.post("/me/avatar", response_model=ImageUploadResponse)
+async def upload_user_avatar(
+    file: UploadFile = File(...),
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)